uniffi_core/src/ffi/rustfuture.rs

```diff
@@ -236,10 +228,11 @@ where
         // Store 1 in `waker_counter`, which we'll use at the end of this call.
         self.wake_counter.store(1, Ordering::Relaxed);
 
-        // `Pin<&mut>`` from our `UnsafeCell`.  `&mut` it is safe, since this is the only reference we
+        // `Pin<&mut>` from our `UnsafeCell`.  `&mut` it is safe, since this is the only reference we
```

Comment:
nano-nit which already existed: extraneous 'it' after `&mut`

Reply:
👍